ui/js/controllers/jobs.js

```diff
@@ -148,7 +148,9 @@ treeherderApp.controller('ResultSetCtrl', [
         };
 
         $scope.showRunnableJobs = function() {
-            $rootScope.$emit(thEvents.showRunnableJobs, $scope.resultset);
+            if ($scope.user.loggedIn) {
```

Comment:
Disabling pointer events on the link only prevents clicks from happening. You could still tab onto the link and trigger it with the keyboard, so this stops that from showing runnable jobs.